dev/acf-export.json

```diff
@@ -1753,6 +1834,162 @@
+                        "name": "track_heading",
+                        "type": "text",
+                        "instructions": "Heading displayed for the track.",
+                        "required": 0,
+                        "conditional_logic": 0,
+                        "wrapper": {
+                            "width": "",
+                            "class": "",
+                            "id": ""
+                        },
+                        "default_value": "",
+                        "placeholder": "",
+                        "prepend": "",
+                        "append": "",
+                        "maxlength": ""
+                    },
+                    {
+                        "key": "field_5f8e6c9195072",
+                        "label": "Track Description",
+                        "name": "track_description",
+                        "type": "textarea",
+                        "instructions": "Short description of the track.",
+                        "required": 0,
+                        "conditional_logic": 0,
+                        "wrapper": {
+                            "width": "",
+                            "class": "",
+                            "id": ""
+                        },
+                        "default_value": "",
+                        "placeholder": "",
+                        "maxlength": "",
+                        "rows": 4,
+                        "new_lines": ""
+                    },
+                    {
+                        "key": "field_5f905f860ec1f",
+                        "label": "Track URL",
+                        "name": "track_url",
+                        "type": "url",
+                        "instructions": "The URL of the track's degree page.",
+                        "required": 0,
+                        "conditional_logic": 0,
+                        "wrapper": {
+                            "width": "",
+                            "class": "",
+                            "id": ""
+                        },
+                        "default_value": "",
+                        "placeholder": ""
+                    }
+                ]
+            },
+            {
+                "key": "field_5f8e6de495076",
+                "label": "Program Tracks Custom Content",
+                "name": "program_tracks_custom_content",
+                "type": "wysiwyg",
+                "instructions": "Custom content displayed at the end of the Program Track section, underneath the individual program track cards.",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "default_value": "",
+                "tabs": "all",
+                "toolbar": "full",
+                "media_upload": 0,
+                "delay": 0
+            },
```

Comment:
We might want to reconsider this set of fields. Program tracks are already defined within WordPress as children posts. The degree importer organizes them this way. You should be able to retrieve them using the [`get_children`](https://developer.wordpress.org/reference/functions/get_children/) function. At that point, we could probably just go ahead and assign any heading/lead/description as custom fields on the track itself, instead of tracking it here in the parent program. This would also make it easier to repeat that data elsewhere, like on the track pages themselves if we wanted an "other tracks" section, for example.